notfound/extension.py

```diff
@@ -192,6 +192,36 @@ def process_doc(self, app, doctree):
         app.env.metadata[app.config.notfound_pagename].update({'orphan': True})
 
 
+def handle_config(app, *args, **kwargs):
```

Comment:
This should match the docstring, as it's a better name:

```suggestion
def handle_ deprecated_config(app, *args, **kwargs):
```